src/util/simplify_expr_struct.cpp

```diff
@@ -206,10 +207,10 @@ simplify_exprt::simplify_member(const member_exprt &expr)
   {
     // Try to look through member(cast(x)) if the cast is between structurally
     // identical types:
-    if(op_type == op.op0().type())
+    if(op_type == to_typecast_expr(op).op().type())
     {
       auto new_expr = expr;
-      new_expr.struct_op() = op.op0();
+      new_expr.struct_op() = to_typecast_expr(op).op();
```

Comment:
We should not do `to_typecast_expr` multiple times.

Reply:
done